src/models/trip.js

```diff
@@ -15,5 +18,9 @@ export default (sequelize, DataTypes) => {
     Trip.belongsTo(models.Location, { foreignKey: 'originId', targetKey: 'id' });
     Trip.belongsTo(models.Location, { foreignKey: 'destinationId', targetKey: 'id' });
   };
+  Trip.afterCreate(({ dataValues }) => {
+    emitter.emit('request-created', dataValues);
+    // TripNotification.sendTripNotification(dataValues);
```

Comment:
Do not leave commented code in your application. Ensure you remove all commented codes